src/sonic-bgpcfgd/setup.py

```diff
@@ -12,7 +12,7 @@
       scripts=['bgpcfgd'],
       entry_points={
           'console_scripts': [
-              'bgpmon = bgpmon:main',
+              'bgpmon = bgpmon_proj.bgpmon:main',
```

Comment:
Suggest naming the new directory `bgpmon` rather than `bgpmon_proj`. Then this line becomes `'bgpmon = bgpmon.bgpmon:main',`.

Reply:
@jleveque Agreed.